### DISCUSSION TITLE
[QSelect] How can I manually hide the dropdown list when specific condition?

### DISCUSSION THREAD

**Discussion by tin80122** [quasarframework/quasar#10000] [General - Components / Directives / etc] [answered]
Hi everyone,

I have an expected behavior:
The user can't open the dropdown select list when number of selected list is over specific number like 4,
Is it possible with QSelect to do that?
Please help me,
Thanks.

**Comment by MilosPaunovic** [comment]
You could use the `disable` prop of `QSelect`
https://quasar.dev/vue-components/select#qselect-api

**Reply by ontwikkelfabriek** [reply]
@tin80122 

Here's a codepen with the `disable` when `number > 4`:
https://codepen.io/ontwikkelfabriek/pen/LYyNJWP

**Reply by tin80122** [reply]
Hi all,
Thank you for replying.
Let me clarify the full expected behaviors.

"The user cannot open the dropdown list when the number of selected tags are over the limit number."

Besides , The user still can delete every tag is in the ￼ input.￼ so after all actions, when the number of selected tags are under the limit number.

Is it possible to implement this behaviors?
Again,thank you for reading.

**Reply by MilosPaunovic** [reply]
It's hard to do this without a codepen or some example code, so if you could provide it, it will be much quicker.

**Reply by tin80122** [reply]
@metalsadman 

oh my god, that's what I exactly wanted behavior!
Thank you so much!

**Comment by metalsadman** [comment, marked as answer]
@tin80122 

https://codepen.io/metalsadman/pen/WNjxzqo?editors=1010
